schemas/rum/error-schema.json

```diff
@@ -177,6 +177,126 @@
+                    "readOnly": true
+                  },
+                  "stack": {
+                    "type": "string",
+                    "description": "Unsymbolicated stack trace of the given thread.",
+                    "readOnly": true
+                  },
+                  "state": {
+                    "type": "string",
+                    "description": "Platform-specific state of the thread when its state was captured (CPU registers dump for iOS, thread state enum for Android, etc.).",
+                    "readOnly": true
+                  }
+                }
+              }
+            },
+            "binary_images": {
+              "type": "array",
+              "description": "Description of the each binary image (native libraries; for Android: .so files) loaded or referenced by the process/application.",
+              "items": {
+                "type": "object",
+                "description": "Description of the binary image (native library; for Android: .so file) loaded or referenced by the process/application.",
+                "required": ["uuid", "name", "is_system"],
+                "properties": {
+                  "uuid": {
+                    "type": "string",
+                    "description": "Build UUID that uniquely identifies the binary image.",
+                    "readOnly": true
+                  },
+                  "name": {
+                    "type": "string",
+                    "description": "Name of the library.",
+                    "readOnly": true
+                  },
+                  "is_system": {
+                    "type": "boolean",
+                    "description": "Determines if it's a system or user library.",
+                    "readOnly": true
+                  },
+                  "load_address": {
+                    "type": "string",
+                    "description": "Library's load address (hexadecimal).",
+                    "readOnly": true
+                  },
+                  "max_address": {
+                    "type": "string",
+                    "description": "Max from the library address range (hexadecimal).",
+                    "readOnly": true
+                  },
+                  "arch": {
+                    "type": "string",
+                    "description": "CPU architecture from the library.",
+                    "readOnly": true
+                  }
+                }
+              },
+              "readOnly": true
+            },
+            "was_truncated": {
+              "type": "boolean",
+              "description": "A boolean value saying if any of the stack traces was truncated due to minification.",
+              "readOnly": true
+            },
+            "meta": {
+              "type": "object",
+              "description": "Platform-specific metadata of the error event.",
+              "properties": {
+                "code_type": {
```

Comment:
The properties listed here are iOS-specific, and it is possible to use `oneOf` if needed, it just doesn't play well with statically-typed languages.